README.md

```diff
@@ -12,7 +12,8 @@ This web application is built to display covid-19 data of countries affected by
 
 ## Live Demo
 
-[View Life](https://duls-travelers-hub.herokuapp.com/)
+- [View video](https://www.loom.com/share/2f173a6ca1594c60a9e760d6d6ef74a3)
+- [View Demo](https://duls-covid-reporter.herokuapp.com/)
   
```

Comment:
- [ ] OPTIONAL
Your Pull Request is indicating that `README.md` was the only file you worked on for this project. Kindly ensure your project have the new features and modifications of the project code **committed to the repository**.